app/components/public/sponsor-list.js

```diff
@@ -5,6 +5,6 @@ const { Component, computed } = Ember;
 
 export default Component.extend({
   sponsorsGrouped: computed('sponsors.[]', function() {
-    return groupBy(orderBy(this.get('sponsors'), 'level'), 'type');
+    return groupBy(orderBy(this.get('sponsors').toArray(), 'level'), 'type');
```

Comment:
This should be 

```js
return groupBy(
  orderBy(
    this.get('sponsors').toArray(), 
    sponsor => sponsor.get('level')
  ), 
  sponsor => sponsor.get('type')
);
```

When dealing with an ember object, make it a point to use the `get` method.